static/locales/en-US.yaml

```diff
@@ -465,12 +458,12 @@ Video:
     Year: Year
     Years: Years
     Ago: Ago
-    Upcoming: Premieres on
-  Published on: Published on
-  Streamed on: Streamed on
-  Started streaming on: Started streaming on
+    Upcoming: Premieres on 
```

Comment:
All these are for https://hosted.weblate.org/translate/free-tube/translations/he/?checksum=dfc62afb07713e1f
Not very elegant, but alas.